src/EnergyPlus/Plant/PlantManager.cc

```diff
@@ -1219,12 +1219,12 @@ void GetPlantInput(EnergyPlusData &state)
                             this_comp.compPtr = EIRPlantLoopHeatPumps::HeatPumpAirToWater::factory(
                                 state, PlantEquipmentType::HeatPumpAirToWaterHeating, CompNames(CompNum));
                             this_comp.Type = PlantEquipmentType::HeatPumpAirToWaterHeating;
-                            this_comp.TypeOf = "HeatPumpAirToWaterHeating";
+                            this_comp.TypeOf = "HeatPumpAirToWater";
```

Comment:
Just delete this line. `this_comp.TypeOf` is set above (line 892) to the string used in the branch input, which is "HeatPump:AirToWater".